### DISCUSSION TITLE
Simplified  Chinese

### DISCUSSION THREAD

**Discussion by xiahao90** [pocketbase/pocketbase#1166] [Q&A]
hello Hope to support Simplified  Chinese

**Comment by ganigeorgiev** [comment]
Could you elaborate a little more?

There are no plans to add translations to the Admin UI before reaching v1, if that's what you are asking.

As for data storage - you should be able to store any unicode character, but currently case-insensitive matches works only for ASCII characters. You could find more info in https://github.com/pocketbase/pocketbase/issues/627.

**Comment by xiahao90** [comment]
我说的是，程序支持多语言，I18n。现在我说的是中文你能看懂吗？
